surfer/viz.py

```diff
@@ -294,17 +294,22 @@ class Brain(object):
         in different viewing panes.
     surf :  geometry name
         freesurfer surface mesh name (ie 'white', 'inflated', etc.)
-    curv : boolean
-        if true, loads curv file and displays binary curvature
-        (default: True)
     title : str
         title for the window
-    cortex : str or tuple
-        specifies how binarized curvature values are rendered.
-        either the name of a preset PySurfer cortex colorscheme (one of
-        'classic', 'bone', 'low_contrast', or 'high_contrast'), or the
-        name of mayavi colormap, or a tuple with values (colormap, min,
-        max, reverse) to fully specify the curvature colors.
+    cortex : str, tuple, or dict
+        Specifies if/how binarized curvature values are rendered.  If
+        binarized curvature values should be rendered it can be
+        either the name of a preset PySurfer cortex colorscheme (one
+        of 'classic' (default), 'bone', 'low_contrast', or
+        'high_contrast'), the name of a mayavi colormap, a 4-tuple with
+        values (colormap, min, max, reverse), or a dictionary with
+        corresponding keys to fully specify the curvature colors. If a
+        color is specified instead, the brain is rendered in that
+        color without displaying the binarized curvature values. A
+        setting of `None` produces a gray brain without binarized
+        curvature.
```

Comment:
I'm a little concerned about how many different options (all with different types) this parameter has. Just reading the keyword argument documentation, I feel a little overwhelmed. I'm not sure what the best solution is, but I think it would be good to think a little bit more about whether this logic can be made a bit simpler.